scalacheck-effect/src/test/scala/org/specs2/scalacheck/effect/ScalaCheckEffectMatchersApiSpec.scala

```diff
@@ -79,17 +79,17 @@ class ScalaCheckEffectMatchersApiSpec extends Specification, ScalaCheckEffect, I
 
  Test parameters can also be specified
    the minimum number of ok tests
-   minTestsOk1
+   $minTestsOk1
 
    the verbosity of a property can be turned on and off
-   verbose1
+   $verbose1
 
   """
-  // def minTestsOk1: Prop =
-  //   propF { (i: Int) => propBoolean(i > 0) ==> (i > 0) }.set(minTestsOk = 50)
+  def minTestsOk1 =
+    propF { (i: Int) => IO(i === i) }.set(minTestsOk = 50)
 
-  // def verbose1: Prop =
-  //   propF { (i: Int) => propBoolean(i > 0) ==> (i > 0) }.verbose
+  def verbose1 =
+    propF { (i: Int) => IO(i === i) }.verbose
```

Comment:
I dropped the implicit conversion to `PropF[IO]`. specs2-scalacheck implements this via `AsResult`, by here we have only `AsExecution` at which point we've erased `F = IO`.

I also tried implementing it like this:
```scala
given ScalaCheckEffectPropertyAsPropF[F[_]]: Conversion[ScalaCheckEffectProperty[F], PropF[F]] = _.propF
```
but then the `.set` parameters stops working. It's probably possible to fix this, but I'm not sure how important it is to be able to convert back to `PropF` so I'm not jumping to chase it down :)

Reply:
Good choice